### ISSUE TITLE
enable favicon

### ISSUE THREAD

**Issue by awitherow** [awitherow/openrefuge#29]
get the logo in as a favicon. gotta do serve-favicon as a node module btw. should be accessible through the static folder.

**Comment by dazulu** [comment]
Why do we need this and not just stick in a normal tiny png favicon tag? I read some of that repo but the caching and everything seems to be more something that huge sites like google or facebook would need to worry about.

**Comment by awitherow** [comment]
Yeah absolutely can do that 👍 i am just a noob and didn't know otherwise.

**Comment by awitherow** [comment]
Joniel is working on a sicknasty logo right now, so gonna wait for him to finish that before we implement.
